go/pkg/flatpak.go

```
@@ -224,286 +180,227 @@ func (f *Flatpak) List() ([]Package, error) {
+	if installed, err := f.InstalledVersion(ctx, name); err != nil {
 		slog.Debug("failed to get installed version", "package", name, "error", err)
 	} else {
 		info.Installed = installed
 	}
 
-	// Get available version from remote
-	out, err := readCmd(f.ctx, "flatpak", "remote-info", "flathub", name).Output()
+	out, err := readOut(ctx, f.r, "flatpak", "remote-info", "flathub", name)
 	if err != nil {
-		return info, nil
+		return info, nil // not available on flathub
```

golangci-lint (2.12.2)

[error] error is not nil (line 296) but it returns nil

(nilerr)